functions/functions.py

```diff
@@ -0,0 +1,55 @@
+import math 
+import random
+
+
+def plus(num1 = 0, num2 = 0) -> int:
+    # The function will gets 2 numbers
+    # The function will return the sum value of the 2 input numbers
+
+    return num1 + num2
+
+
+def greet_name(name_input: str) -> None:
+    # The function gets a string that contains a name
+    # The function will print the str: "Hello {name} great to see you!"
+
+    print(f"Hello {name_input}! Great to see you!")
+
+
+
+def equationroots(a: int, b: int, c: int) -> int: 
+    # The function gets 3 numbers, a , b and c 
+    # The function returns the answer of the quadratic equation
+
+    if a == 0:
+        print("Wrong input! a can't be zero! ")
+    
+    else:
+        discriminants  = b * b - 4 * a * c 
+        sqrt = math.sqrt(abs(discriminants)) 
```

Comment:
You should not name fields with saved names/functions.
The use of abs() does not belong in the formula.